src/auslib/util/versions.py

```diff
@@ -45,6 +45,8 @@ class AncientMozillaVersion(StrictVersion):
 
 class GlobVersionTuple(tuple):
     def __eq__(self, value):
+        if len(value) < len(self):
+            return False
```

Comment:
I noticed that my test script had this, and the PR didn't. This avoids a KeyError.

Reply:
(I realized we don't have to have this, since StrictVersion will always have 3+ len...)